test/helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'mocha/setup'
 require 'fileutils'
 require 'minitest/reporters'
+require 'minitest/spec'
```

Comment:
`minitest/spec` is not used in the rest of Gollum's test suite. So as you can see, the tests included in this PR are very different than the existing test suite.

I am open to changing these tests to better match the existing suite. However, I do feel that spec-style tests are easier to read and write. I'm curious to hear your thoughts on them, and whether it would be worth refactoring some other tests to use the spec DSL.

Reply:
I'm not against using spec format for any newly added tests, as long as it plays well with code coverage! There is no reason for using the older `assert` based tests other than that's how we inherited them. They're more difficult to read, but otherwise function well. I think it would be a lot of effort to migrate them all to spec format, so not sure that's worth it!